spec.bs

```diff
@@ -1022,6 +1109,13 @@ Note: Previously, the `payment` extension allowed for the creation of
       :  <dfn>instrument</dfn> member
       :: The instrument details that were displayed to the user. Only used at authentication time; not registration.
 
+      :  <dfn>browserBoundPubKeyCredParams</dfn> member
+      :: The list of allowed types of credential restricting the types of
+         cryptographic algorithms used for the browser bound key.
+
+         Note: When this member is not present, it defaults to
+             {{PublicKeyCredentialCreationOptions}}.{{PublicKeyCredentialCreationOptions/pubKeyCredParams}}.
```

Comment:
The indenting here seems weird? I don't think it affects the actual rendering, but it's still a little funky.